src/components/Conversion/ConversionNOB.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <section id="nature-of-business-section">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label>Nature of Business</label>
+        <v-chip
+          v-if="hasNatureOfBusinessChanged"
+          id="changed-chip"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          Changed
+        </v-chip>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <div v-if="onEditMode">
+          <p class="ma-0">
+            Provide a brief description of the nature of business (e.g., corner grocery store,
+            automotive repair service, landscaping, etc.).
+          </p>
+          <v-form ref="form" lazy-validation>
+            <v-textarea
+              filled
+              persistent-hint
+              class="mt-5"
+              autocomplete="chrome-off"
+              label="Enter Nature of Business"
+              rows="3"
+              v-model="naicsText"
+              :counter="maxLength"
+              :rules="naicsRules"
+              validate-on-blur
+            >
+            </v-textarea>
+            <div id="nob-confirm-container" class="mb-2">
+              <v-btn large color="primary" id="nob-done-btn" class="mr-2"
+                @click="onSubmitClicked()"
+              >
+                <span>Done</span>
+              </v-btn>
+              <v-btn large outlined color="primary" id="nob-cancel-btn"
+                @click="onCancelClicked()"
+              >
+                <span>Cancel</span>
+              </v-btn>
+            </div>
+          </v-form>
+        </div>
+
+        <div v-if="!onEditMode" class="summary-block d-flex justify-space-between align-center">
+          <span id="naics-summary">{{ naicsSummary }}</span>
+          <v-btn text color="primary" id="nob-change-btn" @click="onChangeClicked()">
+            <v-icon small>mdi-pencil</v-icon>
+            <span>Change</span>
+          </v-btn>
+        </div>
+      </v-col>
+    </v-row>
+  </section>
+</template>
+
+<script lang="ts">
+import { Action, Getter } from 'vuex-class'
+import { Component, Vue } from 'vue-property-decorator'
+import { ActionBindingIF } from '@/interfaces/'
+import { NaicsIF } from '@bcrs-shared-components/interfaces/'
+import { isEqual } from 'lodash'
+
+@Component({})
+export default class NatureOfBusiness extends Vue {
+  @Getter getCurrentNaics!: NaicsIF
+  @Getter hasNatureOfBusinessChanged!: boolean
+
+  @Action setNaics!: ActionBindingIF
+
+  // local variables
+  private onEditMode = false
+  private maxLength = 300
```

Comment:
Technically this is constant, so you can use readonly instead of private.

(Also, technically the template can't see private properties, but at the moment Vue allows it.)

Reply:
I'll remove `maxLength` as it's only used in two places. `onEditMode` is a variable, so setting it to readonly will probably not work.

Reply:
I like the idea of a single definition.

Did you try it as readonly?

Reply:
Yes, I did. It will give me an error because it's a constant: `Cannot assign to 'onEditMode' because it is a read-only property.`

Reply:
Ah, oops, the syntax is different:
`readonly maxLength: 300`

Reply:
Wait a minute.

Max Length is a constant (and can be readonly).

On Edit Mode is a variable that you set True and False. Obviously this cannot be readonly.

Sorry, long day for me.

Reply:
I'll remove maxLength as it's only used in two places.